### PR TITLE
fixed a bug where the kitten function errored out on tinytest, when c…

### PR DESCRIPTION
**Pull request by paulhodor**
…alled with a non-default path; fixed a typo in the help page of kitten

### CODE REVIEW

File: R/pkgKitten.R [hunk 1]
```diff
@@ -131,8 +131,8 @@ kitten <- function(name = "anRpackage",
     file.copy(rdsrc, rdtgt, overwrite=TRUE)
 
     if (puppy && requireNamespace("tinytest", quietly=TRUE)) {
+        tinytest::setup_tinytest(root, verbose=FALSE)
         message(" >> added tinytest support")
-        tinytest::setup_tinytest(name, verbose=FALSE)
```
Comment (eddelbuettel) [inline R/pkgKitten.R:135]: So I messed up `root` vs `name` here?  That's certainly possibe.

Comment (eddelbuettel) [reply R/pkgKitten.R:135]: The things is ... that it works as is. (Expand for log)

<details>

```sh
edd@rob:/tmp$ mkdir kitten
edd@rob:/tmp$ cd kitten/
edd@rob:/tmp/kitten$ kitten.r -h\
> ^C
edd@rob:/tmp/kitten$ kitten.r -h
Usage: kitten.r [-t TYPE] [-b] [-p] [-h] [-x] PACKAGE

-t --type TYPE      type of kitten: plain, rcpp, arma, eigen. [default: plain]
-b --bunny          install roxygen2 documentation example and roxygenize (only for plain)
-p --puppy          invoke tinytest::puppy to set up testing (only for plain)
-h --help           show this help text
-x --usage          show help and short example usage 
edd@rob:/tmp/kitten$ kitten.r -p quickcheck
Creating directories ...
Creating DESCRIPTION ...
Creating NAMESPACE ...
Creating Read-and-delete-me ...
Saving functions and data ...
Making help files ...
Done.
Further steps are described in './quickcheck/Read-and-delete-me'.

Adding pkgKitten overrides.
 >> added .gitignore file
 >> added .Rbuildignore file
 >> added tinytest support
Deleted 'Read-and-delete-me'.
Done.

Consider reading the documentation for all the packaging details.
A good start is the 'Writing R Extensions' manual.

And run 'R CMD check'. Run it frequently. And think of those kittens.


Thank you edd, for showing us some PUPPY LOVE <3

    ,-.___,-.
    \_/_ _\_/
      )O_O(
     { (_) }  W00F!
      `-^-'   
edd@rob:/tmp/kitten$ ls -l quickcheck/tests/tinytest.R 
-rw-rw-r-- 1 edd edd 94 Nov 15 12:49 quickcheck/tests/tinytest.R
edd@rob:/tmp/kitten$ ls -l quickcheck/inst/tinytest/test_quickcheck.R 
-rw-rw-r-- 1 edd edd 56 Nov 15 12:49 quickcheck/inst/tinytest/test_quickcheck.R
edd@rob:/tmp/kitten$ 
```

</details>

How did you call it to trigger an error?  Maybe I get lucky here via `pkgKitten.r` from `litter` ?

Comment (eddelbuettel) [reply R/pkgKitten.R:135]: Ah, I think you just answered in #19.

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+2023-11-15  Paul Hodor  <paul@aurynia.com>
+
+	* R/pkgKitten.R: fixed path in call to tinytest; fixed typo in doc
+	of kitten()
```
Comment (eddelbuettel) [inline ChangeLog:4]: Thank you!

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+
+	* docs/mkdmt-src/: Updated for release 0.2.3
```
Comment (eddelbuettel) [inline ChangeLog:6]: Bit confused about that one but no worries I can nix that later.

The site appears to be current and complete:  https://eddelbuettel.github.io/pkgkitten  (ie News, Changelog)

**Review by eddelbuettel — APPROVED** [review]
Thanks for bug fix!

**Comment by eddelbuettel** [comment]
I think you just force-pushed the change to changelog away again:

image

Try again?

**Comment by paulhodor** [comment]
This is way too much automation for me. I did not realize that changes to my fork get propagated to you. I had made a mistake in the changelog and was working on fixing it. I wasn't aware that you can see all that activity.

**Comment by eddelbuettel** [comment]
> Looks like it went into the same pull request. I guess I should have created a new branch.

(Just playing "git doctor" here: correct.  If you want that, you could do the following:
- create a new branch of your current branch, that is basically for safekeeping
- roll this branch back via eg `git reset --hard HEAD^` should do, I usually reset to a sha1 as I am paranoid)
- force push the rolled back branch to restore this PR
- wait til I merge, pull, then create a new branch
- cherry pick the third commit here and on the safekeep branch into the new branch, PR)

If it is too much work we just merge this. Your call.  It's kinda nice to have two separate PRs but it doesn't matter greatly.

**Comment by paulhodor** [comment]
Let's leave it as is. But thanks for the explanation. I'll remember it for next time.

File: ChangeLog [hunk 4]
```diff
@@ -1,3 +1,17 @@
+2023-11-15  Paul Hodor  <paul@aurynia.com>
+
+	* inst/replacements/test_hello.R: added a test file as a
+	replacement of the default created by setup_tinytest(); it has
+	added tests for the hello() function
+	* inst/replacements/hello.R: better formatting of the "hello" output
+	* R/pkgKitten.R: replace auto-generated test_<name>.R with content
+	of test_hello.R
+
+2023-11-15  Paul Hodor  <paul@aurynia.com>
```
Comment (eddelbuettel) [inline ChangeLog:10]: I'll clean this up post merge.  It's generally an entry per day, not per commit.  So these two will get fused.

**Review by eddelbuettel — APPROVED** [review]

**Pull request merged**